Mathlib/CategoryTheory/FiberedCategory/Cocartesian.lean

```diff
@@ -357,7 +357,7 @@ noncomputable def codomainIsoOfBaseIso {R S S' : 𝒮} {a b b' : 𝒳} {f : R 
     [IsStronglyCocartesian p f' φ'] : b ≅ b' where
   hom := map p f φ h φ'
   inv := @map _ _ _ _ p _ _ _ _ f' φ' _ _ _ _ _ (congrArg (· ≫ g.inv) h.symm) φ
-    (by simp; infer_instance)
+    (by simp only [assoc, Iso.hom_inv_id, comp_id]; infer_instance)
```

Comment:
Isn't `infer_instance` flexible?

Reply:
The linter complained about it, so no. (Feel free to propose making it flexible, and we can discuss.)